server/route/oauth.go

```diff
@@ -0,0 +1,21 @@
+package route
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+// Authorize user
+func Authorize(c *gin.Context) {
+	c.JSON(http.StatusOK, gin.H{
+		"message": "Authorize",
+	})
+}
+
+// Authenticate user
+func Authenticate(c *gin.Context) {
```

Comment:
```go
// Authorize user
func Authorize(c *gin.Context) { ... }

// Authenticate user
func Authenticate(c *gin.Context) { ... }
```

I supposed these route functions above are little ambiguous. I think `Authorize()` should be `GetAuthorization() / GetAuthentication()`. How about this?